src/main/java/uk/gov/laa/ccms/caab/bean/validators/proceedings/ProceedingFurtherDetailsValidator.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.laa.ccms.caab.bean.validators.proceedings;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFlowFormData;
+import uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFormDataFurtherDetails;
+import uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFormDataProceedingDetails;
+import uk.gov.laa.ccms.caab.bean.validators.AbstractValidator;
+
+/**
+ * Validator the further details provided by proceeding flow forms.
+ */
+@Component
+public class ProceedingFurtherDetailsValidator extends AbstractValidator {
+
+  /**
+   * Determines if the Validator supports the provided class.
+   *
+   * @param clazz The class to check for support.
+   * @return {@code true} if the class is assignable from
+   *         {@link uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFlowFormData},
+   *         {@code false} otherwise.
+   */
+  @Override
+  public boolean supports(final Class<?> clazz) {
+    return ProceedingFlowFormData.class.isAssignableFrom(clazz);
```

Comment:
Should this not be ProceedingFormDataFurtherDetails.class, as that is the target of the validator?

Reply:
This one requires access to the proceeding details form data to be able to check if the order type is a required feild